test/specs/simulation.vehicles.spec.js

```diff
@@ -1,19 +1,37 @@
-const { generateRandomVehicles } = require('../../server/simulation/vehicles.js');
+const {generateRandomVehicles, randomBid} = require('../../server/simulation/vehicles.js');
+const {randomCoords} = require('../../server/simulation/random.js');
 
 describe('generateRandomVehicles()', () => {
   test('returns an array with a length same to the vehicleCount argument', () => {
-    expect(
-      generateRandomVehicles(4, {}, 2000)
-    ).toHaveLength(4);
+    expect(generateRandomVehicles(4, {}, 2000)).toHaveLength(4);
   });
 
   test('returns an array of objects specifically matching given props', () => {
-    expect(
-      Array.isArray(generateRandomVehicles(1, {}, 2000))
-    ).toBe(true);
+    expect(Array.isArray(generateRandomVehicles(1, {}, 2000))).toBe(true);
 
-    expect(
-      Object.keys(generateRandomVehicles(1, {}, 2000)[0]).sort()
-    ).toEqual(['id', 'model', 'icon', 'coords', 'rating', 'missions_completed', 'missions_completed_7_days', 'status'].sort());
+    expect(Object.keys(generateRandomVehicles(1, {}, 2000)[0]).sort()).toEqual([
+      'id',
+      'model',
+      'icon',
+      'coords',
+      'rating',
+      'missions_completed',
+      'missions_completed_7_days',
+      'status'
+    ].sort());
+  });
+});
+
+describe('randomBid()', () => {
```

Comment:
May I suggest improving the first part for conciseness and breaking the test itself for readability?

```javascript
describe('randomBid()', () => {
  const getRandomCoords = randomCoords({
    coords: {
      lat: 1,
      long: 1
    },
    radius: 1000
  });

  test('returns an object with a price containing a float', () => {
    const bid = randomBid(getRandomCoords(), getRandomCoords(), getRandomCoords());
    expect(typeof bid.price).toBe('number');
  });
});
```